src/pages/ProductList/ProductList.jsx

```diff
@@ -1,11 +1,86 @@
-import React from 'react'
-import { NavBar } from '../HomePage/HomePageComponents/NavBar/NavBar'
-
+import React from 'react';
+import { NavBar,Footer } from '../HomePage/HomePageComponents';
+import "../../Colours/colours.css";
+import "../ProductList/ProductList.css";
+import {products} from "../../backend/db/products"
```

Comment:
you can import your color css in index.js so it applicable for all component and page
and are why are importing you importing your backed 
you can fetch your backend

Reply:
I am planning to fetch products when I add the complete functionality 👍️

Reply:
I am going to use CSS modules for the whole project and It will make the CSS classes clash and the scoping get distorted. So I didn't mention it in the index.js